lib/iruby/display.rb

```diff
@@ -287,7 +287,11 @@ def format(mime = nil, &block)
       match {|obj| obj.respond_to?(:path) && File.readable?(obj.path) }
       format do |obj|
         mime = MimeMagic.by_path(obj.path).to_s
-        [mime, File.read(obj.path)] if SUPPORTED_MIMES.include?(mime)
+        if RUBY_PLATFORM =~ /mswin(?!ce)|mingw|cygwin/
+          [mime, File.binread(obj.path)] if SUPPORTED_MIMES.include?(mime)
+        else
+          [mime, File.read(obj.path)] if SUPPORTED_MIMES.include?(mime)
+        end
```

Comment:
Well, probably `#binread` is appropriate for binary mime types on all platforms?..